assets/css/style.css

```diff
@@ -0,0 +1,129 @@
+:root {
+  --black: #000;
+  --white: #fff;
+  --light-green: #c5e9da;
+  --light-black: #172b4d;
+  --green: #36b37e;
+  --very-light-black: #505f79;
+  --sm: 14px;
+  --md: 20px;
+  --lg: 45px;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  margin: 0;
+}
+
+.navbar-center ul {
+  display: flex;
+  justify-content: space-evenly;
+  margin: 50px 250px 0 250px;
+}
+
+.navbar-center ul li {
+  list-style: none;
+  font-size: var(--md);
+}
+
+.navbar-center ul li a {
+  text-decoration: none;
+  font-size: var(--md);
+  color: var(--light-black);
+  font-weight: 600;
+}
+
+.mobile {
+  display: none;
+  margin-right: 30px;
+}
+
+.heading {
+  background: url(../img/header-illsutration-mobile@2x.png);
+  background-size: 100%;
+  height: 100vh;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+.network {
+  margin: 0 100px 0 100px;
+}
+
+.network ul {
+  display: flex;
+  justify-content: space-around;
+  padding: 0;
+}
+
+.network ul li {
+  list-style: none;
+}
+
+.network ul li a {
+  text-decoration: none;
+  color: var(--very-light-black);
+  font-size: 30px;
+}
+
+.title h1 {
+  margin: 0 0 0 0;
+  font-size: var(--lg);
+  font-weight: 700;
+  color: var(--light-black);
+  text-align: center;
+}
+
+.title2 {
+  color: var(--green);
+  margin: 0 0 0 0;
+  font-size: var(--lg);
+  font-weight: 700;
+  text-align: center;
+}
+
+.subtitle {
+  font-size: var(--md);
+  text-align: center;
+  margin: 0 25px 0 25px;
+  font-weight: 400;
+}
+
+.button {
+  text-decoration: none;
+  font-size: 40px;
+  color: var(--very-light-black);
+}
+
+.scroll-down {
+  display: flex;
+  justify-content: center;
+}
+
+@media (max-width: 740px) {
```

Comment:
- [ ] Please the use of media queries at this level is not required. Please follow the requirement as it is. The use of media queries will be implemented in the step ahead.

Reply:
Do I have to erase the media query?

Reply:
@OLIPLICHE

Reply:
Yes please